README.md

```diff
@@ -46,111 +47,115 @@ See [action.yml](action.yml) for all options.
 
 To authenticate with the BSR, set the inputs `username` and `token`.
-The `username` and `token` values can be stored as secrets in the repository settings.
+The `username` and `token` values should be stored as secrets in the repository settings.
 The `token` value can be [generated from the Buf Schema Registry UI](https://buf.build/docs/bsr/authentication#create-an-api-token).
 
+> [!IMPORTANT]  
+> Never hardcode these values in the workflow file.
+
 ```yaml
-- uses: bufbuild/buf-action@v0.1.3
+- uses: bufbuild/buf-action@v0.1
   with:
     username: ${{ secrets.BUF_USERNAME }}
     token: ${{ secrets.BUF_TOKEN }}
 ```
 
-Alternatively, you can set the environment variable `BUF_TOKEN`.
-
-```yaml
-- uses: bufbuild/buf-action@v1
-  env:
-    BUF_TOKEN: ${{ secrets.BUF_TOKEN }}
-```
-
 For more information on authentication, see the [Buf Schema Registry Authentication Reference](https://buf.build/docs/bsr/authentication).
 
-#### Security considerations
-
-Always use secrets to store `token` and `username` values.
-Never hardcode these values in the workflow file.
-
-
 ### Summary comment
 
-To help code review feedback the action outputs a GitHub summary of the current check status and comments on the pull requests.
-For each subsequent run the comment updates displaying the latest status:
+<!-- Do we want to avoid posting a comment if no proto files have changed? -->
+The action reports the status of the most recent checks in a comment on each pull request.
 
 ![Comment example showing the GitHub summary](./static/img/comment-example.png "Summary comment example")
 
 To disable the comment, set the input `comment` to `false` and remove the permission `pull_request: write` as this is no longer required.
 
-```yaml
+```diff
 name: Buf CI
-runs-on: ubuntu-latest
+on:
+  push:
+  pull_request:
+    types: [opened, synchronize, reopened, labeled, unlabeled]
+  delete:
 permissions:
   contents: read
-steps:
-  - uses: bufbuild/buf-action@v0.1.3
-    with:
-      comment: false
+- pull-requests: write
+jobs:
+  buf:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: bufbuild/buf-action@v0.1
+        with:
+          version: 1.32.2
+          username: ${{ secrets.BUF_USERNAME }}
+          token: ${{ secrets.BUF_TOKEN }}
++         comment: false
 ```
 
+### Specify input directory
+
+To run the action for inputs not specified at the root of the repository,
+set the input `input` to the path of your `buf.yaml` directory.
 
-### Specify inputs
+Breaking change detection will also need to be configured with the correct value for `breaking_against`.
 
-To run the action for inputs not specified at the root of the repository, set the input `input` to the path of your `buf.yaml` directory.
-Breaking change detection will also be required to be set to include a `subdir` configured to the same input path.
+If you are using a URL, add the `subdir` parameter to match `input`.
```

Comment:
Three sentences as three paragraphs, could maybe be condensed?

Reply:
done in 4ad96d7e8f88e83dcf2139a6147b6e14cee997e8